### PR TITLE
Use single quotes to make regression tests work on Windows

### PR DESCRIPTION
**Pull request by tautschnig**
test.pl cannot cope with double quotes on some Windows-based platforms
(including GitHub's Windows environment).

- [x] Each commit message has a non-empty body, explaining why the change was made.
- n/a Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- n/a The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [x] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- n/a My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- n/a White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Review by martin-cs — APPROVED** [review]
Changes as described.  I will take your word that they are necessary.

File: jbmc/regression/jbmc-json-ui/pointer-simplification/test.desc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 CORE
 test
-'.[] | select(has("result")) | .result | .[] | select(has("trace")) | .trace | .[] | select(has("value")) | .value | select(.name == "pointer") | {has_data: has("data")}' --json-ui --trace
+'.[] | select(has('result')) | .result | .[] | select(has('trace')) | .trace | .[] | select(has('value')) | .value | select(.name == 'pointer') | {has_data: has('data')}' --json-ui --trace
```
Comment (NlightNFotis) [inline jbmc/regression/jbmc-json-ui/pointer-simplification/test.desc]: This seems to be tripping up all the UNIX builds now (ubuntu and osx). 

Is there a way perhaps to have two of these commands, and dispatch based on OS detection at test runtime? I'm not sure we have one, but maybe it wouldn't be hard to come up with one?

If that's not possible/convenient for the time being, maybe this change could be reverted and the test marked as `KNOWNBUG` on windows?

**Review by NlightNFotis — APPROVED** [review]
Hi Michael,

Again many thanks for reactivating the tests on windows. 

Good to go, just one problem with the `jbmc-json-ui/pointer-simplification` test passing the wrong syntax to `jq` on UNIX systems, causing `jq` to throw syntax errors.

File: jbmc/regression/jbmc-json-ui/pointer-simplification/test.desc (hunk 1 above)
Comment (tautschnig) [reply jbmc/regression/jbmc-json-ui/pointer-simplification/test.desc]: Thank you for digging into this one, I was quite confused.

**Pull request merged**